include/miopen/miopen.h

```diff
@@ -573,6 +581,8 @@ typedef enum
     MIOPEN_CONVOLUTION_ATTRIB_DETERMINISTIC =
         1, /*!< Restrict MIOpen convolutions to kernels which produce numerically deterministic
               results. 0 - disabled (default), 1 - enabled >*/
+    MIOPEN_CONVOLUTION_ATTRIB_FP8_ROUNDING_MODE = 2,
+    MIOPEN_CONVOLUTION_ATTRIB_COMPUTE_TYPE      = 3,
```

Comment:
Please add doc - explain what is "compute type".

Reply:
> Please add doc - explain what is "compute type".

GEMM operation is defined as D = alpha * A * B + beta * C, where A*B is dot product.

Reply:
@atamazov  It represents the accumulator type for accumulation in convolution.

Reply:
@JehandadKhan @xinlipn We already have this feature in some other convolutions (e.g. FP16) and in some other primitives (e.g. FP16 pooling). For so far it is hidden, i.e. not exposed in the public API. And I am not sure if it is worth to make it public for them.

If we are going to use this attribute for F8 only, then I recommend either documenting this or (better) renaming the attribute.